lib/OpenQA/Schema/Result/JobGroupParents.pm

```diff
@@ -101,4 +101,32 @@ around 'default_priority' => sub {
     return $self->get_column('default_priority') // OpenQA::Schema::JobGroupDefaults::PRIORITY;
 };
 
+sub child_group_ids {
+    my ($self) = @_;
+
+    my @group_ids;
+    for my $group ($self->children) {
+        push(@group_ids, $group->id);
+    }
```

Comment:
can you replace this with a map?

```perl
my @group_ids = map { $_->id } $self->children;
```

@coolo would call this "pseudo code", i.e. might work, not tested.